libs/portal-layout-styles/src/styles/primeng/sass/variables/theme/_theme_light.scss

```diff
@@ -41,15 +41,16 @@ $errorColor: #b00020;
 //selected state
 $highlightBg: rgba($primaryColor, 0.12);
 $highlightTextColor: $primaryColor;
+$highlightFocusBg: rgba($primaryColor, 0.24) !default;
 
 //scale
 $scaleSM: 0.875;
 $scaleLG: 1.25;
 
 //focus
 $focusOutlineColor: transparent;
-$focusOutline: 1px solid black;
-$focusOutlineOffset: 2px;
+$focusOutline: 0 none;
+$focusOutlineOffset: 0;
```

Comment:
Since we do not use the default outline anymore maybe its good idea to create variables for outlines per component?

Reply:
done